Sophos/sophos_module/trigger_sophos_xdr_query.py

```diff
@@ -91,11 +91,14 @@ def run(self):
     def post_query(self, query: dict):
         # It's the first step of our query treatment.
         # Posting the query
+        self.log(message="Starting the first step of quering the Sophos data lake", level="info")
         response_runQuery = self.client.run_query(json_query=query).json()
 
         query_id = response_runQuery.get("id")
+        self.log(message=f"Getting the query id {query_id}", level="info")
 
         status = ""
+        self.log(message=f"Starting the while loop before finishing", level="info")
```

Comment:
I suggest merging this two logs
```suggestion

        status = ""
        self.log(message=f"Waiting the query {query_id} to complete", level="info")
```